src/components/button/button.scss

```diff
@@ -35,72 +58,67 @@
 
   &.secondary {
     background: transparent;
-    box-shadow: 0px 7px 10px rgba($color-primary-darker, 0.5);
     color: $color-primary;
-    box-sizing: border-box;
-    border: 2px solid $color-primary;
-
-    &:hover, &:focus {
-      box-shadow: 0px 4px 5px rgba($color-primary-darker, 0.6);
-    }
+    @include btn-box-shadow($color-primary-darker);
+    @include btn-borders($color-primary, $color-primary);
 
     &:active {
       background: $color-primary;
-      box-shadow: none;
       color: $color-white;
     }
   }
 
   &.tertiary {
     background: transparent;
     color: $color-primary-darker;
-    box-sizing: border-box;
 
     &:hover, &:focus {
       text-decoration: underline;
     }
-
   }
 
   &.danger {
     background: $color-danger;
-    box-shadow: 0px 7px 10px rgba($color-danger-darker, 0.5);
     color: $color-white;
-
-    &:hover, &:focus {
-      box-shadow: 0px 4px 5px rgba($color-danger-darker, 0.6);
-    }
+    @include btn-box-shadow($color-danger-darker);
+    @include btn-borders($color-danger, $color-danger-darker);
 
     &:active {
       background: $color-danger-darker;
     }
   }
 
   &:disabled {
+    cursor: not-allowed;
     background: $color-grey-light;
-    box-shadow: 0px 0px 20px 3px $color-grey-light;
     color: $color-grey;
+    border: none;
+    @include btn-box-shadow($color-grey-light);
 
     &:hover, &:active {
       transform: none;
-      box-shadow: 0px 0px 20px 3px $color-grey-light;
       background: $color-grey-light;
-      cursor: not-allowed;
     }
   }
 
   &:active {
     text-shadow: none;
   }
 
-  // For icons and svgs
   i, svg {
     vertical-align: middle;
     margin-right: 4px;
   }
 }
 
-.default {
+.small {
+  composes: PHS;
```

Comment:
sorry but what's PHS

Reply:
@crysfel created a way to include paddings and margins.
For mobile first styles:
```
// This adds padding horizontal small
// padding-left: small
// padding-right: small

composes: PHS;
```

**IMPORTANT:**
Inside mediaqueries it does not works. For these cases, we have a mixin called `spacing`
It's all in the file called `_spacing.scss`

Reply:
PHS stands for Padding Horizontal Small, which translate to:

```
.PHS {
  padding-left: 8px;
  padding-right: 8px;
}
```

It's basically a way to keep all our spacing consistent across the app.